File: C#/hello.cs
// Tim Kutcher
// Hello.cs - Hello world in C#.

using System;

namespace HelloWorld
{
    class Hello
    {
        static void SayHello(String name)
        {
            Console.WriteLine(String.Format(
                "Hello, World! My name is {0}.", name));
        }

        static void Main()
        {
            SayHello("C#");
        }
    }
}

File: Java/Hello.java
// Tim Kutcher
// Hello.java - Hello world in Java.

public class Hello {

    public static void hello(String name) {
        System.out.print("Hello, World! ");
        System.out.printf("My name is %s.\n", name);
    }

    public static void main(String[] args) {
        hello("Java");
    }
}
